src/Microsoft.Azure.SignalR.Common/Exceptions/ExceptionExtensions.cs

```diff
@@ -35,5 +37,19 @@ internal static Exception WrapAsAzureSignalRException(this Exception e, Uri base
                     return e;
             }
         }
+
+        internal static Exception WrapAsAzureSignalRException(this Exception e, WebSocketConnectionOptions options)
+        {
+            switch (e)
+            {
+                case WebSocketException webSocketException:
```

Comment:
let's use `StartWith` instead of `Contains` to make the check more straight-forward